_locales/en/messages.json

```diff
@@ -254,5 +254,11 @@
 	},
 	"subscription_expired_error_message": {
 		"message": "Your subscription has expired.  Renew your subscription or downgrade your account to Freemium"
-	}
+	},
+	"add_time_entry": {
+		"message": "Add Time Entry"
+	},
+    "time_greater_than_now": {
```

Comment:
to wcięcie specjalnie czy tylko GH tak pokazuje?

Reply:
poprawiłem wcięcie